### ISSUE TITLE
Virtctl doesnt provide any output on successful start/stop vm

### ISSUE THREAD

**Issue by nellyc** [kubevirt/kubevirt#1221]
**Is this a BUG REPORT or FEATURE REQUEST?**:

> Uncomment only one, leave it on its own line:
>
> /kind bug
/kind enhancement

**What happened**:
virtctl start/stop has no output on success

**What you expected to happen**:
i would expect to get some output, such as 'vm X successfully started/stopped' 

**How to reproduce it (as minimally and precisely as possible)**:
virtctl start/stop vm_name

**Anything else we need to know?**:

**Environment**:
- KubeVirt version (use `virtctl version`):
v0.7.0-alpha.2-dirty (notice 'dirty', expect another issue :))
- Kubernetes version (use `kubectl version`):
- VM or VMI specifications:
- Cloud provider or hardware configuration:
- OS (e.g. from /etc/os-release):
- Kernel (e.g. `uname -a`):
- Install tools:
- Others:

**Comment by itamarh** [comment]
that would require to wait on the result, where the current behaviour is basically "Request to start/stop the VM sent successfully"?

**Comment by nellyc** [comment]
im ok with the current behavior, but would appreciate some output, your suggestion of 'request to...' is good enough for me

**Comment by stu-gott** [comment]
Note to developer:

The commands to start and stop a VM (essentially launch or kill a VirtualMachineInstance) is actually a wrapper around the `kubectl patch` command. The command in question occurs around line 113 of `pkg/virtctl/vm/vm.go`:

```
        _, err := virtClient.VirtualMachine(namespace).Patch(vm.Name, types.MergePatchType, []byte(bodyStr))
```

The current implementation returns an error from the `Run` function, but does not otherwise indicate success.

One possible approach would be to simply add a log message (found in `pkg/log/log.go`), or to simply use "fmt.Printf".

A more elaborate stretch goal would be to honor the original request and block/not return until the VirtualMachineInstance is started or stopped. This would require periodic polling. Something along the lines of:

```
import utilwait "k8s.io/apimachinery/pkg/util/wait"

func checkVM(virtClient kubecli.KubevirtClient) bool {
    ....
}

err := utilwait.PollImmediate(retryInterval, retryLimit, checkVM(vlrtClient, vmName)
if err != nil {
    return fmt.ErrorF("timed out waiting for VM: %v", err)
}
```

Please note, the code above is not complete, it's simply meant to put you on the right track.

HINT: Assuming we started a VirtualMachine named 'vm-cirros', the VirtualMachineInstance of interest would have a label: `kubevirt.io/vm: vm-cirros`.

`pkg/virt-controller/watch/vmi.go` has a function called `filterMatchingPods` which is very close to what we need here (except it's looking for pods):
```
selector, err := v1.LabelSelectorAsSelector(&v1.LabelSelector{MatchLabels: map[string]string{virtv1.DomainLabel: vmi.Name, virtv1.AppLabel: "virt-launcher"}})
```

**Comment by adityaramteke** [comment]
@stu-gott I would like to give a try if no one else

**Comment by BradfordNichols** [comment]
Users coming from OSP and OCP users will not expect a synchronous wait on this kind of operation ( cloud instance or container respectively). Should a synchronous wait for success be an additional flag the use passes ?

**Comment by gouyang** [comment]
The simplest thing is just like #1292, put a message like the request xxx is sent successfully, the message can be improved a little bit like adding "please check the VM status by blabla...".

I vote for  "wait for results" and filtering some message from `virt-handler` pod like:
```
Virtual machine AAA is scheduled to node BBB.
```
If it's using an image, check image existence:
```
Image is not existed...
Pulling image kubevirt/fedora-cloud-registry-disk-demo
VM is ready to start
VM is running
```

**Comment by stu-gott** [comment]
I think @BradfordNichols might have a point here. Perhaps we should make non-blocking the default, and add something like a `--watch` flag to the command to invoke an interactive status update along the lines of what @gouyang articulated.

**Comment by masayag** [comment]
@stu-gott could you have a look at the suggested PR on https://github.com/kubevirt/kubevirt/pull/1307 which added the option to block till the action (start/stop) is completed ?

**Comment by rmohr** [comment]
I think that it is expected that everything is asynchronous in k8s. When I do

```
kubectl.sh scale -n kube-system deployment/kube-dns --replicas=3
deployment.extensions "kube-dns" scaled
```

it also only does a patch. It is not waiting until all three instances are created. I would want us to follow with our message style here. So `virtualmachine "myvm" started` instead of something like `start request for virtualmachine "myvm" issued`.
